src/components/Block/Plugin.vue

```diff
@@ -40,12 +34,20 @@ function getLogoUrl(key) {
           <Icon name="github" class="mr-1" />
           {{ plugin.author }}
         </a>
+        {{
+          $tc('inSpaces', [
+            formatCompactNumber(pluginsSpacesCount?.[plugin.key] ?? 0)
+          ])
+        }}
       </div>
-      {{
-        $tc('inSpaces', [
-          formatCompactNumber(pluginsSpacesCount?.[plugin.key] ?? 0)
-        ])
-      }}
+      <a
```

Comment:
We should center the icon and the text here with `items-center`

Reply:
Done!